config/routes.rb

```diff
@@ -17,8 +17,8 @@ def matches?(request)
   end
 
   resources :concerto_plugins do
-    collection do
-      post :update_gem
+    member do
+      get :update_gem
```

Comment:
@mfrederickson Using a get is probably not ideal here. We're performing a system operation that has a bunch of substantial side effects (https://www.w3.org/2001/tag/doc/whenToUseGet.html).